Formula/o/openvino.rb

```diff
@@ -75,6 +76,16 @@ class Openvino < Formula
     sha256 "c757132e018dd0dd171499ef74fca88b74c5430a20781ec53da19eb7f937ef68"
   end
 
+  resource "openvino-telemetry" do
+    url "https://files.pythonhosted.org/packages/37/dd/675a6349e4b5a1d5bfaab940cd52a70c988099b6f4c1689af1884c49815a/openvino-telemetry-2023.2.1.tar.gz"
+    sha256 "ca2106b84671c6edfdc562c03b1c6dd3434d649a4c2174aa47f4628aa66e660a"
+  end
+
+  resource "packaging" do
+    url "https://files.pythonhosted.org/packages/ee/b5/b43a27ac7472e1818c4bafd44430e69605baefe1f34440593e0332ec8b4d/packaging-24.0.tar.gz"
+    sha256 "eb82c5e3e56209074766e6885bb04b8c38a0c015d0a30036ebe7ece34c9989e9"
+  end
```

Comment:
can someone help with embedding these packages as part of OpenVINO Python API?

Reply:
basically, I don't know how to add `python-*` packages here
We have some OV python specific content and need to add several standard modules on top of it.